src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.java

```diff
@@ -535,6 +535,7 @@
    public abstract AppCommand loadServerHome();
 
    // Build
+   public abstract AppCommand clearBuild();
```

Comment:
Should hide this command when Build pane is not available. Do this in `BuildCommands.java` in `setBuildCommandState()` in same branch that hides `activateBuild`.